k8s/openebs-monitoring/README.md

```diff
@@ -0,0 +1,39 @@
+## Overview
+* [Prometheus](https://github.com/prometheus/prometheus) is a monitoring tool  which is used to monitor OpenEBS volumes. There is already a lot of exporters like [node-exporter](https://github.com/prometheus/node_exporter) are available which can be used to gather the various metrics. 
```

Comment:
There are already a log of exporters like...
remove 'are available'
remove 'the' from the various metrics.